src/Microsoft.TestPlatform.ObjectModel/TraitCollection.cs

```diff
@@ -19,7 +19,7 @@ public class TraitCollection : IEnumerable<Trait>
     internal const string TraitPropertyId = "TestObject.Traits";
     private static readonly TestProperty TraitsProperty = TestProperty.Register(
         TraitPropertyId,
-#if !NET451
+#if !NET462
```

Comment:
this seems wrong. the condition said if not net451 before, so for net462 the condition body should be used, and else case should be removed.

Reply:
Will do in a follow-up PR.